backend/Lingva.WebAPI/Extensions/ServiceExtensions.cs

```diff
@@ -15,7 +15,8 @@ public static class ServiceExtensions
         public static void ConfigureSqlContext(this IServiceCollection services, IConfiguration config)
         {
             string connection = config.GetConnectionString("LingvaConnection");
-            services.AddDbContext<DictionaryContext>(options => options.UseSqlServer(connection), ServiceLifetime.Singleton);
```

Comment:
Do not use any db context in Singleton lifetime scope. All repositories must create as Singleton.
But dbContext make Scoped. What do you think and why?